src/components/panel/git/RepositoryAddModal.tsx

```
@@ -0,0 +1,88 @@
+        <input
+          value={name}
+          onChange={(e) => {
+            setName(e.target.value);
+            if (validationErrorMessage) setValidationErrorMessage("");
+          }}
+          placeholder="레포지토리 이름"
+          className="typo-body6 h-11 w-full rounded-full border border-white bg-white/50 px-4 py-3 text-(--color-text-primary) placeholder:text-(--color-text-tertiary) focus:outline-none"
+          disabled={isPending}
+        />
+
+        <label className="typo-label text-(--color-text-secondary)">URL</label>
```

Biome (2.4.15)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)